e2e/__tests__/__snapshots__/detectOpenHandles.ts.snap

```diff
@@ -13,7 +13,7 @@ This usually means that there are asynchronous operations that weren't stopped i
 exports[`prints out info about open handlers 1`] = `
 "Jest has detected the following 1 open handle potentially keeping Jest from exiting:
 
-  ●  TCPSERVERWRAP
+  ●  DNSCHANNEL,TCPSERVERWRAP
 
       12 | const app = new Server();
       13 |
```

Comment:
I modified the logic here and now it will report all types at the same stack.
Not sure if the prompt message needs to be modified, but I'm not good at English and can't do anything about it.🤦‍♂️

Reply:
Awesome change! We could probably land this on `main`?

Reply:
Yes, I can do this, but the tests in the main branch don't reflect this. I couldn't think of a test example like this for a while. (`DNSCHANNEL` is ignored in main branch)